packages/core/src/tracing/spanBuffer.js

```diff
@@ -179,6 +189,14 @@ exports.addSpan = function (span) {
     return;
   }
 
+  // Process the span, apply any transformations, and implement filtering if necessary.
+  const processedSpan = processSpan(span);
```

Comment:
qs: Putting it here. 

Entry Span http -> Exit Span redis get. 
The feature is called "ignoreEndpoints". Was it ever discussed what will happen to the entry spans?

Reply:
I also have this confusion. We should discuss this!

Reply:
We are ignoring the exit spans, so only the entry span will be displayed in the UI. The exit span will not appear.